frontend/packages/console-dynamic-plugin-sdk/src/extensions/pages.ts

```diff
@@ -15,7 +15,32 @@ export type StandaloneRoutePage = ExtensionDeclaration<
   }
 >;
 
+type ResourcePageCommonProps = {
+  /** See https://reacttraining.com/react-router/web/api/match */
+  match: RouteComponentProps['match'];
+  /** The resource kind scope. */
+  kind: string;
```

Comment:
Shouldn't this be expanded to:
```
 model: {
  group: string,
  version: string,
  kind: string,
 }, 
```

Reply:
I thought we would be passing GVK identifier to the Component.

Reply:
The format of our single GVK string is an internal detail and we can construct it internally given the expanded object.